modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/fragments/group/src/guest-page-collection/fragments/guest-container/index.html

```diff
@@ -1,6 +1,6 @@
 [#if themeDisplay.isSignedIn() && !themeDisplay.getLayout().isTypeControlPanel() && !themeDisplay.getLayout().isSystem() && layoutMode != "edit"]
 	[#assign currentURL = (themeDisplay.getURLCurrent()?keep_before("/guest"))! /]
-	[#assign layoutPath = currentURL?ends_with("/")?then("home", "/home") /]
+	[#assign layoutPath = currentURL + "/home"]
```

Comment:
this seems odd, it may happen to work since this only used on one page, but the previous version looks safer. However both of this line and the one above seem somewhat hacky. It may work for now, but @eliasantosf what issue are you fixing by updating this line?

Reply:
The old version was redirecting the page to the Liferay home page, I just changed it to redirect to the PRM home page

Reply:
Ok for you @ryanschuhler ?